README.md

```diff
@@ -6,7 +6,7 @@
 
 # Please see [Our Documentation](https://kind.sigs.k8s.io/docs/user/quick-start/) for more in-depth installation etc.
 
-kind is a tool for running local Kubernetes clusters using Docker container "nodes".  
```

Comment:
this is intentional, please maintain these. this moves the follow text to a newline in the rendered output.